docs/source/graph-construction/gs-processing/gpartition/sagemaker.rst

```diff
@@ -0,0 +1,133 @@
+===================================
+Running partition jobs on SageMaker
+===================================
+
+Once the :ref:`distributed processing setup<gsprocessing_distributed_setup>` is complete,
+you can use Amazon SageMaker launch scripts to launch distributed processing jobs with AWS resources.
+To demonstrate the usage of GSProcessing on Amazon SageMaker, we will execute the same output in the
+:ref:`GSProcessing Output<gsp-examining-output>` .
+
+Build the docker image for partition jobs on SageMaker
+------------------------------------------------------
+GPartition job on SageMaker uses SageMaker's **BYOC** (Bring Your Own Container) mode.
+
+Step 1: Build a SageMaker-compatible Docker image
+...................................................
+
+.. note::
+    * Please make sure your account has access key (AK) and security access key (SK) configured to authenticate accesses to AWS services.
+    * For more details of Amazon ECR operation via CLI, users can refer to the `Using Amazon ECR with the AWS CLI document <https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html>`_.
+
+First, in a Linux machine, configure a Docker environment by following the `Docker documentation <https://docs.docker.com/get-docker/>`_ suggestions.
+
+In order to use the SageMaker base Docker image, users need to use the following command to authenticate to pull SageMaker images.
+
+.. code-block:: bash
+
+    aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
This account ID only applies for us-east-1, let's add links to all the others: https://docs.aws.amazon.com/sagemaker/latest/dg-ecr-paths/sagemaker-algo-docker-registry-paths.html

Reply:
Let's cite the DLC github page to provide this authentication explanation: https://github.com/aws/deep-learning-containers/blob/master/available_images.md.
There is a section named "Important" that give this command and explanations.